src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -684,5 +684,33 @@ public static partial class Kernel32
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool IsWow64Process(SafeObjectHandle hProcess, out bool Wow64Process);
+
+        /// <summary>
+        /// Creates an anonymous pipe, and returns handles to the read and write ends of the pipe.
+        /// </summary>
+        /// <param name="hReadPipe">
+        /// A pointer to a variable that receives the read handle for the pipe.
+        /// </param>
+        /// <param name="hWritePipe">
+        /// A pointer to a variable that receives the write handle for the pipe.
+        /// </param>
+        /// <param name="lpPipeAttributes">
+        /// A pointer to a SECURITY_ATTRIBUTES structure that determines whether the returned handle can be inherited by child processes. If <paramref name="lpPipeAttributes"/> is NULL, the handle cannot be inherited.
+        /// The <see cref="SECURITY_ATTRIBUTES.lpSecurityDescriptor"/> member of the structure specifies a security descriptor for the new pipe. If <paramref name="lpPipeAttributes"/> is NULL, the pipe gets a default security descriptor. The ACLs in the default security descriptor for a pipe come from the primary or impersonation token of the creator.
+        /// </param>
+        /// <param name="nSize">
+        /// The size of the buffer for the pipe, in bytes. The size is only a suggestion; the system uses the value to calculate an appropriate buffering mechanism. If this parameter is zero, the system uses the default buffer size.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is nonzero.
+        /// If the function fails, the return value is zero. To get extended error information, call <see cref="GetLastError"/>.
+        /// </returns>
+        [DllImport(nameof(Kernel32))]
```

Comment:
But I rebased my branch on this commit and fixed it so if you're ok with it we can close this PR and merge the whole thing when i'm done.